mod/deckorate_attribution/assets/style/reference.scss

```diff
@@ -1,7 +1,36 @@
 .TYPE-reference .RIGHT-Xsubject.card-editor {
-  display: none
+  display: none;
 }
 
-.attribution-alert-link {
-  font-size: small;
+.TYPE-reference .RIGHT-adaptation.card-editor {
+  .form-check {
+    appearance: none;
+    background-clip: padding-box;
+    background-color: #fff;
+    border: 1px solid #ced4da;
```

Comment:
use variable `$gray-400`  instead of hard-coding `#ced4da` (see colors in mod/wikirate/data/real/style.yml)